tests/src/python/test_qgsproviderconnection_postgres.py

```diff
@@ -1045,7 +1045,7 @@ def test_move_raster_with_overviews_to_schema(self):
         # check that overviews were created
         sqlOverviews = """
         SELECT o_table_schema, o_table_name FROM public.raster_overviews 
-        WHERE r_table_schema = 'qgis_test' AND r_table_name = 'raster_for_move' ORDER BY r_table_name;
+        WHERE r_table_schema = 'qgis_test' AND r_table_name = 'raster_for_move' ORDER BY o_table_name;
```

Comment:
this change seems to be unrelated with "fix saving unnamed project in PG DB" ?

Reply:
yes, but the test failed. I can understand why it failed now and not previously. But the test had an error, so I decided to fix it. Should I split it into separate PR?